xapian-applications/omega/index_file.cc

```diff
@@ -1029,6 +1029,9 @@ index_mimetype(const string & file, const string & urlterm, const string & url,
 				d.get_size(), d.get_mtime());
 		return;
 	    }
+
+
+
```

Comment:
Don't randomly add (or remove) blank lines as part of a patch.  If whitespace reformatting changes to existing code are needed (which these aren't) make them as part of a separate patch which only does such reformatting.